about.html

```diff
@@ -82,7 +83,7 @@ <h2 class="text text-center">Health Exhibition and Confress logo</h2>
             trade fair in Ethiopia</p>
         </div>
         <div class="logo-b">
-          <img src="images/pharmac.png" alt="Health Exhibition and Congress">
+          <img src="images/health-logo.png" alt="Health Exhibition and Congress">
         </div>
         <div class="spacing-3"></div>
```

Comment:
- [ ] The footer section is not aligned as is in the given design kindly fix this as well

Your design
![image](https://user-images.githubusercontent.com/81932478/205450310-113781dc-6bd3-47b7-a4d2-e4797bdf8228.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/205450329-5914c9ad-4165-4cd7-877b-dc5cb0106ffd.png)